### PR TITLE
feat: deploy Minibroker for Cloud Foundry

### PR DESCRIPTION
**Pull request by f0rmiga**
Some targets and script to help iterate faster with CF.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -91,8 +91,17 @@ deploy:
 	OUTPUT_CHARTS_DIR="$(OUTPUT_CHARTS_DIR)" \
 	./build/deploy.sh
 
+deploy-cf:
+	IMAGE="$(IMAGE)" \
+	TAG="$(TAG)" \
+	IMAGE_PULL_POLICY="$(IMAGE_PULL_POLICY)" \
+	OUTPUT_CHARTS_DIR="$(OUTPUT_CHARTS_DIR)" \
+	./build/deploy-cf.sh
```
Comment (mook-as) [inline Makefile:99]: `deploy-cf.sh` appears to be nearly identical to `deploy.sh`; can we add options to `deploy.sh` to toggle between the two instead (to ensure that any fixes to one will be applied to the other)?

Not sure why, for example, `deploy-cf.sh` dropped `--recreate-pods`…

Comment (f0rmiga) [reply Makefile:99]: `--recreate-pods` is deprecated, but I agree with you.

Comment (f0rmiga) [reply Makefile]: Done with https://github.com/kubernetes-sigs/minibroker/pull/95/commits/e969dd074a5b5064e98d185ae1fb417f87116f96.

File: build/deploy.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset -o pipefail -o xtrace
```
Comment (mook-as) [inline build/deploy.sh:17]: Do we want the `xtrace` here all the time? I assume that's just left over debugging.  If we really want something like that, it'd probably be:

```suggestion
set -o errexit -o nounset -o pipefail ${XTRACE:+-o xtrace}
```

Comment (f0rmiga) [reply build/deploy.sh:17]: Yeah, leftover. I'll take the suggestion, though.

**Pull request merged**